simpledotorg/simple-server
# Overdue returned to care: Months dashed calculation

**Story card:** [sc-10846](https://app.shortcut.com/simpledotorg/story/10846/add-dynamic-number-of-dashed-lines-based-on-day-of-the-month?vc_group_by=day&ct_workflow=all&cf_workflow=500000031)

## Because

Fix the number of dashed line segments shown in the 'Return to care' chart. Currently it is fixed at 2 dashed segments but should alternate between 1 and 2 dependent on the date of the month

## This addresses

Should show last 2 months dashed (2 lines segments between the last 3 nodes) of the return to care chart on days 1-15 of the current month.
<img width="125" alt="Screenshot 2023-07-04 at 17 56 47" src="https://github.com/simpledotorg/simple-server/assets/9541902/c101dd0a-9519-40e8-b47b-2ab90572abba">

Should show 1 month dashed (last segment between the last 2 nodes) of the return to care chart on days 16-30/31 of the month.
<img width="108" alt="Screenshot 2023-07-04 at 17 57 05" src="https://github.com/simpledotorg/simple-server/assets/9541902/f61d0a36-b803-4e3b-8937-655c0ea296dc">


## Test instructions

If able check using a different date after the 15th (unsure if thats possible).
On the current date (4th) is showing 2 dashed segments.
Otherwise use `const currentDayOfTheMonth = 16;`

- check day-of-the-month
- where > 15 = 1 (1 month dashed)
- where < 16 = 2 (2 months dashed)
